src/main/java/com/tournament/cup/play/WriterFile.java

```diff
@@ -0,0 +1,23 @@
+package com.tournament.cup.play;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+@Repository
+@AllArgsConstructor
+public class WriterFile {
+    public void writer(String str) {
```

Comment:
The repository should be responsible for converting object to string. You should just pass here Squad or Tournament. If you want them all to be serialized to CSV format create base class that knows how to work with CSV format and use it in Squad, Tournament repositories